styleguide/components/21-atoms/field-description/field-description.twig

```diff
@@ -1,5 +1,7 @@
 {% if field_description %}
   <div class="field-description">
     {{ field_description|default(field_description) }}
+
+    <div class="accolade left-top"></div>
```

Comment:
Hier mss via een include werken van het atom?

Reply:
Het atom staat atm invisible en is hard coded op bottom center... Tenzij we dit volledig dynamisch maken wordt dat een beetje lastig :)

Reply:
Ik volg Jeroen, als het een atom is: include en een modifier meegeven.